docs/Version-Support.adoc

```diff
@@ -41,6 +41,8 @@ A commercial support agreement is available from mailto:sales@chronicle.software
 |====
 (a) Provisional dates
 
+From x.24, stable releases will only be supported for at least a year, though they might be supported longer.
```

Comment:
Suggest rephrase more definitely. Something like:
"From x.24, support for stable releases will only be guaranteed for 1 year"